### ISSUE TITLE
Weblate : Some items are translated incorrectly

### ISSUE THREAD

**Issue by WillyB1337** [RigsOfRods/rigs-of-rods#2650]
@AnotherFoxGuy 
First of all, I really do like the platform we are using to make the translations the easiest it can be.

I did notice multiple words seems to be interpreted incorrectly. I first notice the issue here: https://hosted.weblate.org/translate/rigs-of-rods/game/fr/?checksum=796479fb3b852cf8

So, I checked this line: https://github.com/RigsOfRods/rigs-of-rods/blob/c8f4d1b79c7e3679a6b82b38720259e22f9dbb75/source/main/gui/panels/GUI_ConsoleView.cpp#L228

and then checked these lines: https://github.com/RigsOfRods/rigs-of-rods/blob/c8f4d1b79c7e3679a6b82b38720259e22f9dbb75/languages/fr/ror.po#L1061-L1066

For me, these don't make any sense.

Basically, the console filter name for errors would be called **Mirrors**.
It did the same misinterpretation on all the other languages.

Hope this is understandable enough to correct it before release.

**Comment by AnotherFoxGuy** [comment]
This is something that is caused by the `msgmerge` tool with fuzzy matching enabled, what it tries to do is match an old msgid ID and update it to the new one. Fuzzy matching is meant for small changes like updating the string `Message: ` to `Message` without completely removing the old translation.

But this is only an issue for translators since all strings that have the `fuzzy` flag will not be compiled into the .mo file that is being used for the translation in-game
